pkg/config/config.go

```diff
@@ -81,6 +81,12 @@ const (
 
 	// DefaultLogsSenderBackoffRecoveryInterval is the default logs sender backoff recovery interval
 	DefaultLogsSenderBackoffRecoveryInterval = 2
+
+	// DefaultInventoriesMinInterval is the default value for inventories_min_interval, in seconds
+	DefaultInventoriesMinInterval = 300 // 5min
```

Comment:
Extra-nit: since you've mentioned "in seconds" above, the value could be `5*60` to make the minutes amount appear without an extra comment (please ignore if you don't like the idea 😅)

Reply:
Sure, that makes sense.